include/supercluster.hpp

```diff
@@ -20,16 +22,43 @@ namespace supercluster {
 using namespace mapbox::geometry;
 
 struct Cluster {
-    point<double> pos;
-    std::uint32_t num_points;
+    const point<double> pos;
+    const std::uint32_t num_points;
     std::uint32_t id;
     std::uint32_t parent_id = 0;
     bool visited = false;
 
-    Cluster(point<double> pos_,
-            std::uint32_t num_points_,
-            std::uint32_t id_)
-        : pos(std::move(pos_)), num_points(num_points_), id(id_) {
+    Cluster(const point<double> pos_, const std::uint32_t num_points_, const std::uint32_t id_)
+        : pos(pos_), num_points(num_points_), id(id_) {
+    }
+
+    feature<double> toGeoJSON() const {
+        const double x = (pos.x - 0.5) * 360.0;
+        const double y =
+            360.0 * std::atan(std::exp((180.0 - pos.y * 360.0) * M_PI / 180)) / M_PI - 90.0;
+        return { point<double>{ x, y }, getProperties(),
+                 std::experimental::make_optional(identifier(static_cast<std::uint64_t>(id))) };
+    }
+
+    property_map getProperties() const {
+        property_map properties{};
+        properties["cluster"] = true;
+        properties["cluster_id"] = static_cast<std::uint64_t>(id);
+        properties["point_count"] = static_cast<std::uint64_t>(num_points);
```

Comment:
Let's use emplace here, or construct the `property_map` directly with an initializer_list